src/background.js

```diff
@@ -1,20 +1,20 @@
 async function sendCommandToTab(command, tab) {
-  // Let's have multiple methods to find the right button, because Spotify
-  // decided front-end development was more exciting with randomized CSS
-  // classes... but not for all UI elements of course. Yay, consistency!
-  function findAndClick(command) {
-    // https://github.com/mantou132/Spotify-Lyrics/issues/94
+  // Let's have multiple methods to find the right button, because Spotify
```

Comment:
Apologies for this - I have my IDE set up to highlight comments followed by an emspace in a different colour, which I use for intended info-type comments like this so that I can visually scan whether I've left any commented-out test code in a file. I apparently just staged the whole file on that commit rather than chunking changes, sorry.

Reply:
No worries!